packages/react-topology/src/components/TopologyView/examples/Topology.md

```diff
@@ -69,3 +70,13 @@ Within each node in your `NODES` array, you can set `data` to include additional
 
 ```ts file='./TopologyCustomNodeDemo.tsx'
 ```
+
+### Pan/Zoom
+
+To create a demo that supports pan/zoom:
+
+1. Import `withPanZoom` into your file.
+2. In your component factory, call `withPanZoom` on the `GraphComponent` it returns.
+
```

Comment:
To allow the user to pan and zoom on the graph, you can use the `withPanZoom` utility when returning the graph component in the componentFactory: `withPanZoom(GraphComponent)`

The component should except a `panZoomRef` property. This should be the `ref` on  the surface element for the group, the element that will except the drag or mouse wheel events.

Alternatively, you can use the `usePanZoom` hook within the component to retrieve the `panZoomRef` property.

If you are using the provided `GraphComponent`, this property is accepted and will be handled appropriately.